src/tap/mod.rs

```diff
@@ -52,6 +52,10 @@ pub struct Opts {
     /// Whether to reconnect if the underlying API connection drops. By default, tap will attempt to reconnect if the connection drops.
     #[arg(short, long)]
     no_reconnect: bool,
+
+    /// Specifies a duration (in ms) to sample logs (e.g. specifying 10000 will sample logs for 10 seconds then exit)
+    #[arg(short, long)]
+    duration: Option<u64>,
```

Comment:
Could we call this `duration_ms`? This is per https://github.com/vectordotdev/vector/blob/master/docs/specs/configuration.md#naming

Reply:
Sounds good, just updated